codalab/model/bundle_model.py

```diff
@@ -2332,8 +2332,9 @@ def get_user_time_quota_left(self, user_id, user_info=None):
         time_used = user_info['time_used']
         return time_quota - time_used
 
-    def get_user_parallel_run_quota_left(self, user_id):
-        user_info = self.get_user_info(user_id)
+    def get_user_parallel_run_quota_left(self, user_id, user_info=None):
```

Comment:
Won't ` if not user_info` always be true since nowhere is passing the argument?

Reply:
Yes. That's the case when we want to make a database query.

Reply:
Then how is it different from the original code? Like isn't the change just like
`
call()
`
to
`
if true:
    call()
`
?
Am I missing something?

Reply:
The point is to make less database query so that its performance will be improved.

Reply:
The parameter is passing from bundle_manager.py

Reply:
But this PR doesn't change bundle_manager right🧐